### PR TITLE
Create discord-rare-drop-notificater

### PR DESCRIPTION
**Pull request by MasterKenth**
Plugin for automatically posting detailed loot notifications and screenshots to Discord using webhooks when receiving a rare/unique drop. Configurable rarity/value threshold. Activities (Barrows, CoX etc.) and pets supported.

example

### CODE REVIEW

**Comment by Nightfirecat** [comment]
This does not compile.

```
/tmp/external-plugin.s4QhGWEJ/repo/src/main/java/com/masterkenth/RarityTables.java:31: error: cannot find symbol
import static java.util.Map.entry;
^
  symbol:   static entry
  location: interface
  where K,V are type-variables:
    K extends Object declared in interface Map
    V extends Object declared in interface Map
/tmp/external-plugin.s4QhGWEJ/repo/src/main/java/com/masterkenth/RarityChecker.java:61: error: cannot find symbol
      log.warn("No event table for '" + eventName + "'");
      ^
  symbol:   variable log
  location: class RarityChecker
```

**Comment by dekvall** [comment]
You need to [pin your dependencies](https://github.com/runelite/plugin-hub#third-party-dependencies).

**Comment by MasterKenth** [comment]
> You need to [pin your dependencies](https://github.com/runelite/plugin-hub#third-party-dependencies).

I've followed those steps and there is a `gradle/verification-metadata.xml` in the repo, unless I'm missing something else? Re-running the command gives me no file changes to commit.

**Comment by dekvall** [comment]
Apparently i missed it, sorry about that. That's correct then.

File: plugins/discord-rare-drop-notificater [hunk 1]
```diff
@@ -0,0 +1,2 @@
+repository=https://github.com/MasterKenth/discord-rare-drop-notificater.git
+commit=d19ad18af3d7154c337d43d43260de61abdba59e
```
Comment (abextm) [inline plugins/discord-rare-drop-notificater:2]:
```suggestion
commit=d19ad18af3d7154c337d43d43260de61abdba59e
warning=This plugin submits the ID of monsters you kill, items you receive, and your IP address to a 3rd party website not controlled or verified by the RuneLite Developers.
```

**Pull request merged**